proto/motor/v1/request.proto

```diff
@@ -112,3 +107,17 @@ message UpdateBucketRequest {
   sonrio.sonr.bucket.BucketRole role = 5;
   repeated sonrio.sonr.bucket.BucketItem content = 6;
 }
+
+message QueryWhereIsRequest {
+  string creator = 1;
+  string did = 2;
+}
+
+message QueryBucketContentBySchemaRequest {
```

Comment:
@ntindle @mcjcloud @brokecollegekidwithaclothingobsession @joshLong145  We need to figure out a better naming convention for query methods collectively lol